templates/event.php

```diff
@@ -31,122 +33,163 @@
 							<a href="<?php echo esc_url( get_author_posts_url( $contributor->ID ) ); ?>"><?php echo esc_html( get_the_author_meta( 'display_name', $contributor->ID ) ); ?></a>
+							<?php
+							if ( ( $user_is_attending && $attendee->is_host() ) || current_user_can( 'manage_options' ) ) :
+								$_attendee_repo = new Attendee_Repository();
+								$_attendee      = $_attendee_repo->get_attendee( $event_id, $contributor->ID );
+								echo '<form class="add-remove-user-as-host" method="post" action="' . esc_url( gp_url( "/events/host/$event_id/$contributor->ID" ) ) . '">';
+								if ( $_attendee->is_host() ) :
+									echo '<input type="submit" class="button is-primary remove-as-host" value="Remove as host"/>';
+								else :
+									echo '<input type="submit" class="button is-secondary convert-to-host" value="Convert to host"/>';
+								endif;
+								echo '</form>';
+							endif;
+							?>
 						</li>
 					<?php endforeach; ?>
 				</ul>
 			</div>
-	<div class="event-details-stats">
-		<h2><?php esc_html_e( 'Stats', 'gp-translation-events' ); ?></h2>
-		<table>
-			<thead>
-			<tr>
-				<th scope="col">Locale</th>
-				<th scope="col">Translations created</th>
-				<th scope="col">Translations reviewed</th>
-				<th scope="col">Contributors</th>
-			</tr>
-			</thead>
-			<tbody>
-			<?php /** @var $row Stats_Row */ ?>
-			<?php foreach ( $event_stats->rows() as $_locale => $row ) : ?>
-			<tr>
-				<td title="<?php echo esc_html( $_locale ); ?> "><a href="<?php echo esc_url( gp_url_join( gp_url( '/languages' ), $row->language->slug ) ); ?>"><?php echo esc_html( $row->language->english_name ); ?></a></td>
-				<td><?php echo esc_html( $row->created ); ?></td>
-				<td><?php echo esc_html( $row->reviewed ); ?></td>
-				<td><?php echo esc_html( $row->users ); ?></td>
-			</tr>
-		<?php endforeach ?>
-			<tr class="event-details-stats-totals">
-				<td>Total</td>
-				<td><?php echo esc_html( $event_stats->totals()->created ); ?></td>
-				<td><?php echo esc_html( $event_stats->totals()->reviewed ); ?></td>
-				<td><?php echo esc_html( $event_stats->totals()->users ); ?></td>
-			</tr>
-			</tbody>
-		</table>
-	</div>
-	<div class="event-projects">
-		<h2><?php esc_html_e( 'Projects', 'gp-translation-events' ); ?></h2>
-		<ul>
-			<?php foreach ( $projects as $project_name => $row ) : ?>
-			<li class="event-project" title="<?php echo esc_html( str_replace( ',', ', ', $row->locales ) ); ?>">
-				<a href="<?php echo esc_url( gp_url_project( $row->project ) ); ?>"><?php echo esc_html( $project_name ); ?></a> <small> to
-				<?php
-				foreach ( explode( ',', $row->locales ) as $_locale ) {
-					$_locale = \GP_Locales::by_slug( $_locale );
-					?>
-					<a href="<?php echo esc_url( gp_url_project_locale( $row->project, $_locale, 'default' ) ); ?>"><?php echo esc_html( $_locale->english_name ); ?></a>
+		<?php endif; ?>
+		<?php if ( ! empty( $attendees ) ) : ?>
```

Comment:
We should only show it to hosts.

Reply:
I am not sure about this. Another sites, like meetup.com, show all attendees in public. Take a look at this [Elementor meetup](https://www.meetup.com/es-ES/elementor-espana-meetup-group/events/299627457/).

It is ok for me to show the attendees' list in public, and it is interesting when you look for information about the event if you are going to be alone or with more people in the event.

![image](https://github.com/WordPress/wporg-gp-translation-events/assets/1667814/6539ae02-d4e1-4532-8ea1-9e9e794456be)

Reply:
I understand your point. I really want to encourage contribution and only show people on the page when they have contributed, not only when they have made a free pledge.

Maybe we could show the list of attendees only before the event starts and then hide it?

Reply:
This is a better approach. What about showing them just until the event finishes?
Imagine a contributor day, starting at 9:00 am and finishing the same day at 4:00 pm. If a user arrives at 9:30, it will be interesting to show her the attendees. And after 4:00 pm, we only show them the contributors. In fact, it could be interesting to ping some users who have forgotten to show up at the event.

Reply:
Yeah but for that wouldn't it be enough for the host to see the attendees? Probably worth testing out with real life events as it's easy enough to change (even during the event).

I think we should try to avoid that some people just always attend all events and they will always appear on all event attendees lists. That's my reasoning behind hiding the attendees list.

Reply:
Yes, it would be enough for the host to see the attendees. But sometimes you decide to go to an event based on who is there, or if there are a few or a lot of people, so I think until the event is finalized, we should show this information to everyone. Then hide the attendees who have not contributed, and that's it.

It is easy to change it, so we can test it like this and if we have any complaints, we can change it according to the new requirements.

Reply:
I tend to agree that only showing the user on the page if they contribute could encourage more contribution.

At the same time, I think there's also value in displaying the list of attendees. Showing that an event has attendees could be encouragement to other people to also attend.

One solution that would address both these needs would be to display the list of attendees in another page (we could also consider introducing tabs on the event details page). This page/tab could then be linked from the main event page, for example below the "Attend" button.

Reply:
While considering a public list of attendees, there should be an option for attendees to choose not to be listed on the public list.